frontend/src/components/pages/TournamentRegistrations/TournamentRegistrations.tsx

```diff
@@ -43,9 +51,10 @@ export const TournamentRegistrations = () => {
     PlayerProfileDto[] | undefined
   >(undefined);
   const [players, setPlayers] = useState<PlayerProfileDto[]>([]);
-  const [registeredPlayers, setRegisteredPlayers] = useState<
-    { playerId: string }[]
-  >([]); //TODO: Fetch already registered players
+  const [registeredPlayersIds, setRegisteredPlayersIds] = useState<string[]>(
+    []
+  );
+  const [isUpdate, setIsUpdate] = useState(false);
```

Comment:
O kurde, a to co :D ?

Reply:
używany do update graczy zarejestrowanych na turniej :-)

Reply:
jeszcze w useEffect na końcu zamiast pustej arrayjki [] dam [registeredPlayersIds] to wtedy renderuje nieskończoną ilość razy ... wgl całość tournament registrations i tak się renderuje kilka razy, nie wiem czy tak powinno być

Reply:
@Szambelan Co to robi tutaj?

Reply:
> @Szambelan Co to robi tutaj?

Que? no przecież napisałem wyżej, więc jakiej odpowiedzi oczekujesz :-p ?

Reply:
A, jakos nie pokazaly mi się te odpowiedzi. 
Obczaje to bo nie podoba mi sie to isUpdated.

Reply:
ja się walnie console.log np. na registeredPlayers to na starcie wyprintuje to z 4-5 razy, więc nie wiem czy coś jednak tego tam za dużo, czy przy tylu useState i 2x useEffect to normalne ? To nie powinno być jakoś przez Reacta ogarnięte - wrzucone do jednego worka i wyrenderowane tylko raz, wszystko?

Reply:
no dobra, ale to jest praktycznie to samo :-P czemu tamto było złe :-) ?

Reply:
No nie masz jakiegoś isUpdated :P Musiałbyś to dodawać w każdym komponencie. 
Tak to mówisz dokładnie do ma się zrobić reloadRegisteredPlayers.
No i jeszcze jedno - to jest źle, bo useEffect musi zalezec od tournamentId.
Gdybys np. wybieral z listy turnieje i pokazywal zapisanych na dany turniej to to Ci się nie przeładuje teraz.